### DISCUSSION TITLE
App changes HTTP link in README to HTTPS when opening

### DISCUSSION THREAD

**Discussion by radiant64** [community/community#69027] [Mobile]
### Select Topic Area

Bug

### Body

Hi,

I've got a repository with a README.md file, from where I link to a page on my personal website (which is served over HTTP; I don't deal with any secrets/user data, and am an unlikely target to be MitM-hijacked). When I open the link from within the GitHub app, it seemingly changes the http:// protocol specifier to https://, resulting in a connection refused message. If I open the link from GitHub in my web browser on the same phone, the http:// link works as expected.

**Comment by taki-on** [comment]
Hi, @radiant64! Thank you for reaching out to us. I tested the scenario on iOS, and it correctly opens a link to `http` in Safari. Do you see the issue on an Android device? It'd be great if you could share the repo where we can reproduce the issue 🙇

**Reply by radiant64** [reply]
Thanks for the prompt reply! Yes, I am on Android (12, for what it's worth). The repo in question is [Cobnetic](https://github.com/radiant64/cobnetic), and the link I see the issue with is specifically the one pointing at [dtepr](http://martinevald.net/software/dtepr.html).

**Reply by radiant64** [reply]
(And I have the same issue with the link when I put it here, unsurprisingly.)

**Reply by taki-on** [reply]
Sorry for late! I will pass the issue to our Android folks to have a look! Thank you very much for sharing those links. They should be very helpful!

**Comment by vivek09thakur** [comment]
It sounds like you're encountering an issue with the GitHub app automatically changing the HTTP link to HTTPS, which is causing a connection refused error. Thi is likely due to security settings within the GitHub app, which may automatically attempt to use HTTPS for links to enhance security.

**Reply by radiant64** [reply]
There is no such setting in the app.
